src/frontends/onnx/frontend/src/input_model.cpp

```diff
@@ -148,12 +149,69 @@ std::shared_ptr<Function> InputModelONNX::convert() {
 }
 
 // Editor features
+bool InputModelONNX::is_coorect_place(const ov::frontend::Place::Ptr& place) const {
+    if (const auto tensor = std::dynamic_pointer_cast<PlaceTensorONNX>(place)) {
+        return m_editor->is_correct_tensor_name(tensor->get_names()[0]);
+    } else if (const auto op = std::dynamic_pointer_cast<PlaceOpONNX>(place)) {
+        return m_editor->is_correct_and_unambiguous_node(op->get_editor_node());
+    } else if (const auto input_edge = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(place)) {
+        if (auto tensor = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(input_edge->get_source_tensor())) {
+            return m_editor->is_correct_tensor_name(tensor->get_names()[0]);
+        }
+    } else if (const auto output_edge = std::dynamic_pointer_cast<PlaceOutputEdgeONNX>(place)) {
+        if (auto tensor = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(output_edge->get_target_tensor())) {
+            return m_editor->is_correct_tensor_name(tensor->get_names()[0]);
+        }
+    }
+    return false;
+}
+
+std::string InputModelONNX::get_place_name(const ov::frontend::Place::Ptr& place) const {
+    if (const auto tensor = std::dynamic_pointer_cast<PlaceTensorONNX>(place)) {
+        return place->get_names()[0];
+    } else if (const auto op = std::dynamic_pointer_cast<PlaceOpONNX>(place)) {
+        return op->get_editor_node().m_node_name;
+    } else if (const auto input_edge = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(place)) {
+        if (auto tensor = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(input_edge->get_source_tensor())) {
+            return tensor->get_names()[0];
+        }
+    } else if (const auto output_edge = std::dynamic_pointer_cast<PlaceOutputEdgeONNX>(place)) {
+        if (auto tensor = std::dynamic_pointer_cast<PlaceInputEdgeONNX>(output_edge->get_target_tensor())) {
+            return tensor->get_names()[0];
+        }
+    }
+    throw std::invalid_argument("Incorrect place type");
+}
+
 void InputModelONNX::override_all_outputs(const std::vector<Place::Ptr>& outputs) {
```

Comment:
Try to apply the equivalent solution to `override_all_inputs`

Reply:
Done.